src/Defaults/Trigger/User/UserEmailChanged.php

```diff
@@ -0,0 +1,96 @@
+<?php
```

Comment:
You didn't apply the [CONTRIBUTING](https://github.com/BracketSpace/Notification/blob/develop/CONTRIBUTING.md) guidelines (commit naming)